lms/djangoapps/commerce/models.py

```diff
@@ -1,3 +1,25 @@
 """
-This file is intentionally empty. Django 1.6 and below require a models.py file for all apps.
+Configuration model for ecommerce service (Otto).
```

Comment:
This docstring describes the class below. The module-level docstring should describe the module. It can be as simple as "Commerce-related models."